From 3b5d090e6ddd6f04c55e6fca5478b14bdceda91f Mon Sep 17 00:00:00 2001
From: Lukas Fleischer <lfleischer@lfos.de>
Date: Sun, 22 Jun 2025 09:50:53 -0400
Subject: [PATCH] Replace __super with super()

Support for the __super property has been removed in urwid commit
9378442 [1].

[1] https://github.com/urwid/urwid/commit/9378442

Signed-off-by: Lukas Fleischer <lfleischer@lfos.de>
---
 urwidtrees/widgets.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/urwidtrees/widgets.py b/urwidtrees/widgets.py
index f600d3e..c4e5923 100644
--- a/urwidtrees/widgets.py
+++ b/urwidtrees/widgets.py
@@ -114,7 +114,7 @@ class TreeBox(WidgetWrap):
         self._outer_list = ListBox(self._walker)
         if focus is not None:
             self.set_focus(focus)
-        self.__super.__init__(self._outer_list)
+        super().__init__(self._outer_list)
 
     # Widget API
     def get_focus(self):
-- 
2.50.0

